app/models/git_repository.rb

```diff
@@ -47,14 +47,15 @@ def commit_from_ref(git_reference)
 
   # @return [nil, tag-sha or tag]
   def fuzzy_tag_from_ref(git_reference)
-    return unless ensure_local_cache!
+    return unless update_local_cache!
     capture_stdout 'git', 'describe', '--tags', git_reference
   end
 
   # @return [nil, tag]
```

Comment:
some docs on `fuzzy tags returns v76 or v76-20-gedd45c2 and we only want it in the first case, prefer to use git tag --points-at but it is only available in git 2.x+`